build_tools/rename_zip.js

```diff
@@ -0,0 +1,25 @@
+/*
+ *   Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
```

Comment:
Suggest add some comment to describe what does this file do.

If using plugin version in artifact name is a standard among all plugin, I would suggest update the upstream plugin-helper code to avoid having this file in all plugins.

Reply:
This is a hack added while I wait for them to figure out a better way...update this comment to TODOs in this PR